scoreboard.pl

```diff
@@ -650,10 +679,30 @@ ($)
         print "        in local_update_cache, post=$post\n" if $verbose;
 
         foreach my $file (@existing) {
-            if ( -e $file . "_Totals.html" ) {next;}
-            if ( $post == 1 ) {
+            if ( -e $file . "_Totals.html" || $post == 1 ) {
                 print "        Prettifying $file.txt\n" if($verbose);
-                Prettify::Process ("$file.txt");
+
+                my $diffRev = '';
+                if (defined $builds{$buildname}->{SUBVERSION_CHECKEDOUT_OPENDDS} &&
+                    !($builds{$buildname}->{SUBVERSION_CHECKEDOUT_OPENDDS} =~ /None/)) {
+                    $diffRev = $builds{$buildname}->{SUBVERSION_CHECKEDOUT_OPENDDS};
+                }
+                else {
+                    $diffRev = 'None';
+                }
+                my $diffRoot = $builds{$buildname}->{DIFFROOT};
+                my $link = '';
+                my $linktarget = '';
+                if (defined $main::opt_n) {
+                    $linktarget = "target=\"_blank\""
+                }
+                # If we have a diff revision, and a diffroot URL, create a link
+                if (($diffRev !~ /None/) && ($diffRoot)) {
+                  my $url = $diffRoot . $diffRev;
+                  $link = "<a href='$url' $linktarget>$diffRev</a>";
+                }
+
+                Prettify::Process ("$file.txt", $buildname, $use_build_logs, $link, $directory . "/" . $log_prefix);
```

Comment:
```suggestion
                Prettify::Process ("$file.txt", $buildname, $use_build_logs, $link, "$directory/$log_prefix");
```

To be consistent with concatenation vs. interpolation on the same line.